static/src/javascripts/projects/common/modules/commercial/dfp-api.js

```diff
@@ -519,7 +523,9 @@ define([
         },
         allAdsRendered = function (slotId) {
             if (slots[slotId] && !slots[slotId].isRendered) {
+                slots[slotId].isLoading = false;
                 slots[slotId].isRendered = true;
+                slots[slotId].slot = null;
```

Comment:
I think it is a great habit to try and tidy things up as we go. I just wouldn't do them all in one pull request, because then you end up with situations where your fix introduces other bugs and you don't have any time to test things thoroughly.

The last thing I would want is to discourage anyone from thinking outside the narrow bounds of the ticket.